webapp/components/create_comment.jsx

```diff
@@ -75,7 +75,7 @@ export default class CreateComment extends React.Component {
             showPostDeletedModal: false,
             enableAddButton,
             showEmojiPicker: false,
-            emojiPickerEnabled: Utils.isFeatureEnabled(Constants.PRE_RELEASE_FEATURES.EMOJI_PICKER_PREVIEW)
```

Comment:
Can you get rid of `state.emojiPickerEnabled` and just use `window.mm_config.EnableEmojiPicker === 'true'` inline in the `Textbox` component?